src/app/configs/data/shortcuts.xml

```diff
@@ -699,7 +699,7 @@
     </SC>
   <SC>
     <key>voice-assignment-all-in-instrument</key>
-    <seq>Ctrl+Alt+0</seq>
+    <seq>Ctrl+Alt+0</seq><!-- conflicts with 'stretch+' on German keyboards! -->
```

Comment:
Not sure we really need a default shortcut for this at all.
Then again with the advent of system locks in 4.5, the strtch shortcuts lost a lot of their importance...